.github/workflows/build_and_test.yml

```diff
@@ -508,22 +508,45 @@ jobs:
           --build-arg WORKER_MODE_ARG=${{ matrix.mode }} --build-arg SGX_COMMERCIAL_KEY=enclave-runtime/intel_sgx.pem --build-arg SGX_PASSFILE=enclave-runtime/passfile.txt --build-arg SGX_PRODUCTION=1 --build-arg ADDITIONAL_FEATURES_ARG=${{ matrix.additional_features }} --build-arg SGX_MODE=${{ matrix.sgx_mode }}
           -f build.Dockerfile .
 
-      - name: Save released teeracle
+      - name: Build Worker Client & Run Cargo Test
```

Comment:
Cargo test is no longer executed as part of the docker build since https://github.com/integritee-network/worker/pull/1480. So we can remove this from the name.